portal-wire-protocol.md

```diff
@@ -153,38 +157,41 @@ selector     = 0x05
 content      = Union[connection_id: Bytes2, content: ByteList[2048], enrs: List[ByteList[2048], 32]]
 ```
 
-* `connection_id`: Connection ID to set up a uTP stream to transmit the requested data.
-    * Connection ID values **SHOULD** be randomly generated.
-*  `content`: byte string of the requested content.
-    * This field **MUST** be used when the requested data can fit in this single response.
-* `enrs`: List of byte strings, each of which is an RLP encoded ENR record.
-    * The list of ENR records **MUST** be closest nodes to the requested content that the responding node has stored.
-    * The set of derived `node_id` values from the ENR records **MUST** be unique.
-    * The ENR record of the requesting & responding node **SHOULD** be filtered out of the list.
+- `connection_id`: Connection ID to set up a uTP stream to transmit the requested data.
+  - Connection ID values **SHOULD** be randomly generated.
+- `content`: byte string of the requested content.
+  - This field **MUST** be used when the requested data can fit in this single response.
+- `enrs`: List of byte strings, each of which is an RLP encoded ENR record.
+  - The list of ENR records **MUST** be closest nodes to the requested content that the responding node has stored.
+  - The set of derived `node_id` values from the ENR records **MUST** be unique.
+  - The ENR record of the requesting & responding node **SHOULD** be filtered out of the list.
 
 If the node does not hold the requested content, and the node does not know of any nodes with eligible ENR values, then the node **MUST** return `enrs` as an empty list.
 
-Upon *sending* this message with a `connection_id`, the sending node **SHOULD** *listen* for an incoming uTP stream with the generated `connection_id`.
+Upon _sending_ this message with a `connection_id`, the sending node **SHOULD** _listen_ for an incoming uTP stream with the generated `connection_id`.
 
-Upon *receiving* this message with a `connection_id`, the receiving node **SHOULD** *initiate* a uTP stream with the received `connection_id`.
+Upon _receiving_ this message with a `connection_id`, the receiving node **SHOULD** _initiate_ a uTP stream with the received `connection_id`.
```

Comment:
unneeded change as mentioned above.

Reply:
The _ and - changes are required by the ethereum/EIPs linter. We should review and merge Pipers PR before this one though @kdeme

Reply:
ok, fair enough. Same linter should be setup on this repo then also.

Reply:
https://github.com/ethereum/portal-network-specs/issues/321

Reply:
Very good. I will be happy if I can help in this matter